packages/ui/src/hooks/useAccountBaseFromAccountList.tsx

```diff
@@ -1,15 +1,26 @@
 import { useMemo } from 'react'
 import { AccountBaseInfo } from '../components/select/GenericAccountSelection'
 import { useAccounts } from '../contexts/AccountsContext'
+import { useAccountNames } from '../contexts/AccountNamesContext'
 
-export const useAccountBaseFromAccountList = () => {
+interface Params {
+  withAccountsFromAddressBook: boolean
+}
+
+export const useAccountBaseFromAccountList = (params?: Params) => {
```

Comment:
can we use here object destruction?
```ts
({ withAccountsFromAddressBook = false }: Params) => 
```

Reply:
This is what I usually prefer too, but in this case, it doesn't allow Param to be undefined. I wanted to keep `Params | undefined` so as not to change anything in the `useAccountBaseFromAccountList` calls that are happening around. And because we can't destructure an undefined object, and afaik there's also no way to say something like `Param | undefined` and at the same time set `{ withAccountsFromAddressBook = false }` as default value.. that's why I went for this.